server.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+
+app.get("/ping", (req, res) => {
+  res.send("Hello world!");
+});
+
+app.listen(3000);
```

Comment:
Consider adding error handling middleware to your express server. This can be done by adding the following code after your routes: `app.use((err, req, res, next) => { res.status(500).send('Something broke!') })` [important]